### ISSUE TITLE
validate callbacks does not work

### ISSUE THREAD

**Issue by sinsoku** [zdennis/activerecord-import#405]
`before_validation` does not work in v0.18.0.

## Test code to reproduce

```rb
# test/models/topic.rb
before_validation -> { errors.add(:title, :invalid) if title == 'invalid' }

# test/import_test.rb
it "should run callbacks" do
  assert_no_difference "Topic.count" do
    Topic.import columns, [["invalid", "Jerry Carter"]], validate: true
  end
end
```

see also: https://github.com/sinsoku/activerecord-import/commit/96fb256a28bad3fbba58754c8fe7fe867b44d7e6

**Comment by jkowens** [comment]
Thank you for discovering this issue! I'm working on a fix and will hopefully have an updated release ready soon.

**Comment by jkowens** [comment]
This should be fixed in the v0.18.1 release!

**Comment by sinsoku** [comment]
Thanks! 👍
